rebar.config

```diff
@@ -20,6 +25,7 @@
         {mimerl, "~>1.1"},
         {certifi, "2.5.2"},
         {metrics, "1.0.1"},
+        {parse_trans, "3.3.0"},
```

Comment:
Is this needed? parse_trans is coming with certifi normally

Reply:
it's fine i will commit with it

Reply:
I just meant to be explicit, since `hackney` started depending directly on it.